core/src/main/java/com/amplifyframework/devmenu/DevMenuLogsFragment.java

```diff
@@ -38,6 +39,10 @@ public DevMenuLogsFragment() { }
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_logs, container, false);
+        View logsView = inflater.inflate(R.layout.fragment_logs, container, false);
+        // Display the logs (if any).
+        TextView logsText = logsView.findViewById(R.id.logs_text);
+        logsText.setText(DeveloperMenu.singletonInstance(getContext()).getLogs());
```

Comment:
This is probably not in scope for this PR, but it would be great if the log menu updated in real time, instead of just `onCreateView`.   Is that possible?

Reply:
It might be possible using either Android's ObservableList to store the logs in each PersistentLogger or making PersistentLogger observable.